cloud/aws/templates/aws_oidc/alarms.tf

```diff
@@ -119,3 +119,51 @@ resource "aws_cloudwatch_metric_alarm" "memory_swap_usage_too_high" {
     DBInstanceIdentifier = data.aws_db_instance.civiform.id
   }
 }
+
+// Connection Count
+resource "aws_cloudwatch_metric_alarm" "connection_count_anomalous" {
+  count               = var.rds_create_anomaly_alarm ? 1 : 0
+  alarm_name          = "rds-${data.aws_db_instance.civiform.id}-anomalousConnectionCount"
+  comparison_operator = "GreaterThanUpperThreshold"
+  evaluation_periods  = var.rds_alarm_evaluation_period
+  threshold_metric_id = "e1"
+  alarm_description   = "Anomalous database connection count detected. Something unusual is happening."
```

Comment:
**suggestion (blocking):** how might we make this description actionable for a civic entity IT responder?

Reply:
Updated!